src/sa_web/static/js/views/dataset-form-view.js

```diff
@@ -0,0 +1,192 @@
+/*globals _ Spinner Handlebars Backbone jQuery Gatekeeper */
+
+var Shareabouts = Shareabouts || {};
+
+(function(S, $, console){
+  S.DatasetFormView = S.PlaceFormView.extend({
+    events: {
+      'submit form': 'onSubmit',
+      'change input[type="file"]': 'onInputFileChange',
+      'click .category-btn-label-clickable': 'onCategoryChange',
+      'click .category-menu-hamburger': 'onExpandCategories',
+      'click input[data-input-type="binary_toggle"]': 'onBinaryToggle'
+    },
+    initialize: function(){
+      var self = this;
+      // keep track of relevant catgory & dataset info as user switches among categories
+      this.selectedCategory = null;
+      this.selectedDatasetId = null;
+      this.priorDatasetId = null;
+      this.selectedDatasetSlug = null;
+      this.priorModelCid = null;
+      S.TemplateHelpers.overridePlaceTypeConfig(this.options.placeConfig.items,
+        this.options.defaultPlaceTypeName);
+      S.TemplateHelpers.insertInputTypeFlags(this.options.placeConfig.items);
+
+      // attach collection listeners
+      for (var collection in this.collection) {
+        this.collection[collection].on('add', self.setModel, this);
+      }
+    },
+    render: function(category, category_selected){
+      // Augment the model data with place types for the drop down
+
+      if (category != undefined) {
+        S.TemplateHelpers.insertInputTypeFlags(this.options.placeConfig.categories[category].fields);
+      }
+
+      var data = _.extend({
+        place_config: this.options.placeConfig,
+        selected_category: this.options.placeConfig.categories[category],
+        category_selected: category_selected || false,
```

Comment:
Can we change the `category_selected` key and parameter to `is_category_selected`? I think it would be a little more clear that it's a boolean, and avoid confusion with `selected_category`.